src/template/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "cdk",
+  "version": "0.1.0",
+  "bin": {
+    "cdk": "bin/cdk.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "format": "prettier --write \"{lib,bin}/**/*.ts\"",
+    "cdk": "cdk",
+    "lint": "eslint lib/** bin/** --ext .ts --no-error-on-unmatched-pattern"
+  },
+  "devDependencies": {
+    "@aws-cdk/assert": "1.74.0",
+    "@guardian/eslint-config-typescript": "^0.4.1",
+    "@types/jest": "^26.0.10",
+    "@types/node": "10.17.27",
+    "@typescript-eslint/eslint-plugin": "^4.8.1",
+    "@typescript-eslint/parser": "^4.8.1",
+    "aws-cdk": "1.74.0",
+    "eslint": "^7.14.0",
+    "eslint-config-prettier": "^6.15.0",
+    "eslint-plugin-eslint-comments": "^3.2.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-prettier": "^3.1.4",
+    "jest": "^26.4.2",
+    "prettier": "^2.2.0",
+    "ts-jest": "^26.4.4",
+    "ts-node": "^9.0.0",
+    "typescript": "~4.0.5"
+  },
+  "dependencies": {
+    "@aws-cdk/assert": "1.74.0",
```

Comment:
This dependency is declared in `devDependencies` too - should this be duplicated?

Reply:
Good spot! I'm 99% sure I copied this from one of the other CDK projects so we can probably remove it somewhere else too.